src/Sentry.Maui/Internal/Extensions.cs

```diff
@@ -0,0 +1,70 @@
+namespace Sentry.Maui.Internal;
+
+internal static class Extensions
+{
+    public static void AddBreadcrumbForEvent(this IHub hub,
+        object? sender,
+        string eventName,
+        string? type,
+        string? category,
+        Action<Dictionary<string, string>>? addExtraData)
+        => hub.AddBreadcrumbForEvent(sender, eventName, type, category, default, addExtraData);
+
+    public static void AddBreadcrumbForEvent(this IHub hub,
+        object? sender,
+        string eventName,
+        string? type = null,
+        string? category = null,
+        BreadcrumbLevel level = default,
+        Action<Dictionary<string, string>>? addExtraData = null)
+    {
+        var data = new Dictionary<string, string>();
+        if (sender is Element element)
+        {
+            data.AddElementInfo(element, null);
+        }
+
+        addExtraData?.Invoke(data);
+
+        var message = sender != null ? $"{sender.GetType().Name}.{eventName}" : eventName;
+        hub.AddBreadcrumb(message, category, type, data, level);
+    }
+
+    public static void AddElementInfo(this IDictionary<string, string> data, Element? element, string? property)
+    {
+        if (element is null)
+        {
+            return;
+        }
+
+        var typeName = element.GetType().Name;
+        var prefix = (property ?? typeName) + ".";
+
+        if (property != null)
+        {
+            data.Add(property, typeName);
+        }
+
+        // The element ID seems to be mostly useless noise
+        //data.Add(prefix + nameof(element.Id), element.Id.ToString());
+
+        if (element.StyleId != null)
+        {
+            // The StyleId correlates to the element's name if one is set in XAML
+            // TODO: Is there a better way to get this?
+            data.Add(prefix + "Name", element.StyleId);
+        }
+
+        if (element is ITitledElement { Title: { } } titledElement)
+        {
+            // TODO: Scrub PII ?
+            data.Add(prefix + nameof(titledElement.Title), titledElement.Title);
+        }
+
+        if (element is IText { Text: { } } textElement)
+        {
+            // TODO: Scrub PII ?
```

Comment:
We're not adding PII scrubbing rules in SDK but this can be scrubbed on BeforeBreadcrumb if there's a risk.
Since we're adding this OOTB, it's less of an issue as if we added on a patch release or something. So folks can see what goes OOTB and add their own PII stripping rules on the backend, or BeforeBreadcrumb if needed

Reply:
Honestly, I think I should just remove it.  It's too risky, IMHO.

Reply:
Or put it behind a feature flag perhaps?